.github/workflows/trivy.yml

```diff
@@ -0,0 +1,72 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Trivy vulnerability scanner
```

Comment:
Since we need a built hale studio to scan in `rootfs` mode I think it would make more sense to integrate the scan into a workflow that anyway already creates a hale studio artifact.

Reply:
sure. But this can be done as well after we understood which tool we like the most, right?

Reply:
Do you refer to trivy and codeql?
In my view both serve different purposes so both should be included.

Reply:
Also in my view Trivy and CodeQL have different focuses, they complement each other very well and using them both we can cover both dependency vulnerabilities and code-level issues,  but I thought that trivy is favourite and we care less about the rest :-)

Reply:
@emanuelaepure10 So can we proceed in that you adapt the trivy scan to scan the product, or what would be your plan?